pom.xml

```diff
@@ -266,7 +266,9 @@
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-javadoc-plugin</artifactId>
             <configuration>
-              <additionalparam>-Xdoclint:none</additionalparam>
+              <additionalOptions>
+                <additionalOption>-Xdoclint:none</additionalOption>
```

Comment:
there's an option for doclint `<doclint>none</doclint>`
https://maven.apache.org/plugins/maven-javadoc-plugin/javadoc-mojo.html#doclint